### PR TITLE
Use gradle 4.4 with android build tool 3.1.3

### PR DESCRIPTION
**Pull request by XiaowenVC**
Hello @PLNech 

In my last pull request https://github.com/algolia/instantsearch-android/pull/83 I forgot to update android build tool. Therefore the line of code which activates data binding v2 does not work. So here is a new PR to fix it. Sorry for the mistake 
c.f. https://developer.android.com/topic/libraries/data-binding/start#preview-compiler

### CODE REVIEW

File: instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java [hunk 1]
```diff
@@ -525,7 +525,7 @@ public void addFacet(@NonNull String attribute, boolean isDisjunctive, @Nullable
      * */
     @SuppressWarnings({"WeakerAccess", "unused"}) // For library users
     public Searcher setFacet(@NonNull String attribute, boolean isDisjunctive) {
-        if (isDisjunctive && disjunctiveFacets.contains(attribute)) {
+        if (isDisjunctive && !disjunctiveFacets.contains(attribute)) {
```
Comment (PLNech) [inline instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java]: Whoops, I should have catched that :sweat_smile:

Comment (XiaowenVC) [reply instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java]: That's my bad.

Comment (PLNech) [reply instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java]: Noprob, I'll merge and publish an update.

Comment (PLNech) [reply instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java]: Noprob, I'll merge and publish an update. One request though, would you mind rebasing your commits to follow the [conventional changelog](http://conventionalcommits.org/) approach? This way the changelog gets generated and we don't have to manually edit it after the release. Something like:

- `chore(gradle): Use gradle 4.4 with android build tool 3.1.3` 
- `fix(Searcher): setFacet logic error`

**Pull request merged**